src/services/sessions.js

```diff
@@ -1755,7 +1770,7 @@ module.exports = class SessionsHelper {
 	static async isTimeSlotAvailable(id, startDate, endDate, sessionId) {
 		try {
 			const sessions = await sessionQueries.getSessionByUserIdAndTime(id, startDate, endDate, sessionId)
-			if (!sessions) {
+			if (!sessions || sessions.startDateResponse.length < process.csv.SESSION_CREATION_MENTOR_LIMIT) {
```

Comment:
Where are you getting this '**startDateResponse**' from ?
Also why is it handled in this function which is specific to isTimeSlotAvailable()

Reply:
@nevil-mathew We are querying it from sessions table and we have a custom response for the start and end Date
So we are introducing the new flow where 5 sessions can be created for one mentor for same time . where he can accept one and reject rest .